packages/page-alerts/src/sass/_module.scss

```diff
@@ -0,0 +1,121 @@
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// page-alerts module
+// Use page alerts to notify users of important information and state changes to the page.
+//
+// Content:
+// - Generic page alert styling
+// - Generic page alert icon position styling
+// - Page alert error
+// - Page alert warning
+// - Page alert success
+// - Page alert info
+// - Page alert screen-reader only
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+@import '_globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Page alert styling
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+/**
+ * Generic page alert styling.
+ */
+.uikit-page-alerts {
+	position: relative;
+	margin: uikit-space( 1 ) 0;
+	padding: uikit-space( 0.75 ) uikit-space( 0.75 ) uikit-space( 0.5 ) 72px;
```

Comment:
This defeats the purpose to be able to omit the icon and have a thin border. (The text will still be indented)
I had a change of heart. We should always have the same size border and when you add the div with the icon the icon is displayed in the middle inside the border. That way the looks is always the same and we never have an empty div anywhere. I'll make those changes now :)